README.md

```diff
@@ -38,6 +39,12 @@ If your sketch has a Dependency, add the `[SketchDependsOn(Type serviceType, Typ
 
 By adding this attribute along with the given dependency type and implementation type, Sketch will create the necessary dependencies for your sketch to run. Note that dependencies don't have to be a sketch's direct dependency. A dependency will be injected for **any** object instantiated in the sketch.
 
+#### Overriding Dependencies
+
+From version 0.1.0 onwards, users can add more specific dependency injection for subclasses of sketches, providing greater flexibility when extending abstract sketches. This is done using the new `SketchDependsOverrideAttribute`.
+
+This feature might be particularly useful if you're using Flume with many sketches with shared behaviour, where varied behaviour from inherited or abstract sketches is desired.
```

Comment:
Should give a concrete example of usage instead of an abstract suggestion.

Reply:
New issue created so somebody can circle back when there's more time.